src/hooks/api/advert/p2p-advert/useSubscribeInfo.ts

```diff
@@ -0,0 +1,26 @@
+import { useMemo } from 'react';
+import { useSubscribe } from '@deriv-com/api-hooks';
+
+/**
+ * A custom hook that subscribes to the `p2p_advert_info` endpoint.
+ *
+ * @example const { data, subscribe } = useSubscribeInfo();
+ */
+const useSubscribeInfo = () => {
```

Comment:
can we remove useAdvertInfo then?

Reply:
we can't, we don't really subscribe to all adverts, sometimes we just want to fetch the data from them. Hence why i just added a separate hook for BuySellForm. However we can investigate which ones we could use this hook and replace useAdvertInfo